meta-ostro-xt/recipes-devtools/checksec/checksec_1.5-1.bbappend

```diff
@@ -0,0 +1,3 @@
+SRC_URI[md5sum] = "075996be339ab16ad7b94d6de3ee07bd"
+SRC_URI[sha256sum] = "77b8a7fd9393d10def665658a41176ee745d5c7969a4a0f43cefcc8a4cd90947"
```

Comment:
In this case the .sh comes from the source mirror it seems. We should probably get rid of that mirrored copy and fix ostro-os-xt master with: https://github.com/ostroproject/ostro-os/commit/913b474c47dd83e2d4ea564464c8e21186867dd7